hockey-score/src/app/teams/team/team.component.spec.ts

```diff
@@ -1,14 +1,23 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { TeamComponent } from './team.component';
+import { NO_ERRORS_SCHEMA, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SharedModule } from 'src/app/shared/shared.module';
+import { MatCardModule } from '@angular/material';
 
 describe('TeamComponent', () => {
   let component: TeamComponent;
   let fixture: ComponentFixture<TeamComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ TeamComponent ]
+      declarations: [ TeamComponent ],
+      schemas: [NO_ERRORS_SCHEMA, CUSTOM_ELEMENTS_SCHEMA],
+      imports: [CommonModule,
+        SharedModule,
+        MatCardModule
+      ]
     })
     .compileComponents();
```

Comment:
The problem with this test is the following: 
it consumes data by @Input() and at the moment of component creation in the test method, it does not have 'team' defined. so it the reason of error in the html where this property is used for rendering.
the same problem appears in PlayerComponent.

there are several ways to fix it. see the link with some examples below

https://medium.com/better-programming/testing-angular-components-with-input-3bd6c07cfaf6

Reply:
fix @Input()